code/modules/admin/verbs/adminhelp.dm

```diff
@@ -146,3 +147,46 @@
 	if(config.useircbot)
 		shell("python nudge.py [msg] [msg2]")
 	return
+
+
+
+
+/client/verb/mentorhelp(msg as text)
+	set category = "Admin"
+	set name = "Mentorhelp"
+
+	if(say_disabled)
+		usr << "<span class='danger'>Speech is currently admin-disabled.</span>"
+		return
+
+	if(prefs.muted & MUTE_ADMINHELP)
+		src << "<span class='danger'>Error: Admin-PM: You cannot send adminhelps (Muted).</span>"

```

Comment:
Maybe this should say "mentorhelps"?
Also, why not mention that it's a mentorhelp somewhere in the message the admins see?
Also, >being bwionked because of wanting to know how to do something.  Can admins be given a means of replying without playing that blood curdling sound?